dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -1061,7 +1061,7 @@ BEGIN
 
     /* For IDD_LOG_OFF_FANCY */
     IDS_LOG_OFF_DESC "Vă închide programele și vă încheie sesiunea ReactOS."
-    IDS_SWITCH_USER_DESC "Permite altui utilizator să fie logat, în timp ce programele dumneavoastră rămân deschise.\r\n\r\n(Puteți de asemenea, să schimbați utilizatorii apăsând tasta cu sigla Windows + L.)"
+    IDS_SWITCH_USER_DESC "Permite altui utilizator să fie logat, în timp ce programele dumneavoastră rămân deschise.\r\n\r\n(Puteți de asemenea, să schimbați utilizatorii apăsând tasta cu sigla ReactOS + L.)"
```

Comment:
The Windows here means the Windows-key on the keyboard, and therefore most likely should not be changed to ReactOS, as there is no ReactOS-key on any keyboard of the world. It may be changed to "Win + L" or "WIN + L" though, if you would like to make this more clear for the next one.

Reply:
Fixed!